cypress/e2e/basic-functionalities/compose/index.cy.js

```diff
@@ -0,0 +1,10 @@
+import './namespace/Create.cy'
+import './module/Create.cy'
+import './record-page/Create.cy'
+import './record-page/Edit.cy'
+// import './record/index.cy'
```

Comment:
remove

Reply:
Gonna keep the test since it may be used in the future